src/components/ComponentCard.jsx

```diff
@@ -1,11 +1,18 @@
 import Link from 'next/link';
 
-export default function ComponentCard({ name, slug, image }) {
+export default function ComponentCard({
+  name, slug, imageOpen, imageClosed,
+}) {
   return (
     <Link href={slug} className="component-link">
       <div className="component-card">
         <span className="component-card__header">{name}</span>
```

Comment:
If we end up keeping the images that already have the span headings, we should delete this, and the corresponding SCSS for it.

Reply:
This is what I'm seeing: 

<img width="1254" alt="Screen Shot 2022-12-14 at 9 21 06 AM" src="https://user-images.githubusercontent.com/69602589/207624870-0dce7c73-b8af-4389-a30a-40f4740fa334.png">

Reply:
I'll take a look re: the screenshots below--some of those look really strange w/ the sizing! I didn't see that yesterday but want to see if I replicate that on my end now too

Reply:
Ah, I see--very odd things are happening at mobile view. I don't think I can fully address these until the updated images have come in, but can figure out why disclosure has disappeared completely on mobile view. That should not be happening!